src/Row.php

```diff
@@ -216,6 +217,10 @@ public function getDoctrineEntityProperty(mixed $item, mixed $key): mixed
 			return (string) $value;
 		}
 
+		if (is_object($value) && $value instanceof BackedEnum) {
```

Comment:
```suggestion
		if (interface_exists(\BackedEnum::class) && $value instanceof BackedEnum) {
```

Reply:
@juniwalk I prefer something like this.

Reply:
No problem, I will add it.

Reply:
Why do you check it, if `$value` is object with `is_object` ?

Reply:
Oh I wish I knew :D I usually don't do that.